src/status_im/data_store/groups.cljs

```diff
@@ -0,0 +1,24 @@
+(ns status-im.data-store.groups
+  (:require [status-im.data-store.realm.groups :as data-store]
+    [re-frame.core :refer [dispatch]])
+  (:refer-clojure :exclude [exists?]))
+
+(defn- normalize-contacts
```

Comment:
IMO, it will look better this way:
```clojure
(defn normalize-contacts [item]
  (update item :contacts vals))

(defn get-all []
  (map normalize-contacts (data-store/get-all-as-list))
```
, but this is not required. Also we have the same function in `status-im.data-store.chats`, so maybe better to move to some common ns.